source/dsl/DangerDSL.ts

```diff
@@ -16,14 +17,19 @@ export interface DangerDSLType {
    *  The GitHub metadata.
    */
   readonly github: Readonly<GitHubDSL>
+
+  /**
+   * Danger utils
```

Comment:
I don't think it should be needed, I expect the `DangerUtilsDSL` to show up here 👍